internal/handler/handler.go

```diff
@@ -31,9 +31,13 @@ type dataRequestV1 struct {
 func New(l log.Logger, c cache.Cacher, wt transport.WrapperFunc, cfg *config.Config) http.HandlerFunc {
 	kubeconfigPath := cfg.KubeconfigPath
 	tenantAPIGroups := cfg.Mappings
-	matcher := cfg.Opa.Matcher
 	debugToken := cfg.DebugToken
 
+	matcherSkipTenants := make(map[string]bool)
```

Comment:
No need to store any values in the map. This map really serves as a set; all we care about is membership. You can tell because you never store `false` in the map.
```suggestion
	matcherSkipTenants := make(map[string]struct{})
```

Reply:
I picked `bool` instead of `struct{}` on purpose because it makes the [line checking membership](https://github.com/observatorium/opa-openshift/pull/10/files#diff-0097c8e59074fdb549fd2466fb349b2814b208df005f9ccf97b14568d51a5f0aR144) much cleaner to read. When using `struct{}` as a map value it would become:

```go
if _, skip := skipTenants[tenant]; matcher == "" || skip {
	return ""
}
```

Which I would probably split up into two if-conditions for readability. Making the map value `bool` eliminates this issue. Changing the map value to `struct{}` for performance reasons also does not matter here as the map is only initialized once and is small.

Reply:
this is the convention we use in all Observatorium and Thanos repositories because using a bool for the value is simultaneously overspecified and ambiguous.